tests/fixtures/with_multiple_readme_files/pyproject.toml

```diff
@@ -0,0 +1,19 @@
+[tool.poetry]
+name = "my-package"
+version = "0.1"
+description = "Some description."
+authors = [
+    "Wagner Macedo <wagnerluis1982@gmail.com>"
```

Comment:
Can you replace author information by Sébastiens or some synthetic data, please? We want to avoid introducing any new real-world information in test data.

Reply:
Makes sense, I liked this.

Done!